testing/scripts/seldon_e2e_utils.py

```diff
@@ -42,6 +43,27 @@ def wait_for_shutdown(deploymentName, namespace):
         ret = run(f"kubectl get -n {namespace} deploy/{deploymentName}", shell=True)
 
 
+def get_deployment_names(seldonDeploymentName, namespace, attempts=20, sleep=5):
+    for _ in range(attempts):
+        ret = run(
+            f"kubectl get -n {namespace} seldondeployment {seldonDeploymentName} -o yaml",
```

Comment:
Here you can use sdep for the seldondeployment

Reply:
Will do.

Reply:
Done.